### DISCUSSION TITLE
FanControl GUI issue

### DISCUSSION THREAD

**Discussion by VeilSide47** [Rem0o/FanControl.Releases#2213] [General]
It seems that expanded cards are not saved. So every time I open FanControl I have to manually expand them again. Saving a profile with them being expanded doesn't work either. 
Is this normal behavior by design? Would be nice to be able to find the GUI when you open up FanControl exactly the way you had it before.

**Comment by Rem0o** [comment]
That state is not serialized/saved, as it would trigger a "Configuration has changed" prompt everytime you leave a card opened.
